exec/java-exec/src/test/java/org/apache/drill/exec/store/TestImplicitFileColumns.java

```diff
@@ -110,4 +111,20 @@ public void testImplicitColumnsForParquet() throws Exception {
         .go();
   }
 
+  @Test // DRILL-4733
+  public void testMultilevelParquetWithSchemaChange() throws Exception {
+    try {
+      test("alter session set `planner.enable_decimal_data_type` = true");
```

Comment:
When I run this query without decimal data type enabled, drill tell me to turn it on. Probably it's connected with data inside the dataset.